src/auth0/handlers/guardianPhoneFactorSelectedProvider.js

```diff
@@ -9,10 +9,24 @@ export const schema = {
       enum: constants.GUARDIAN_PHONE_PROVIDERS
     }
   },
-  required: [ 'provider' ],
   additionalProperties: false
 };
 
+const isFeatureUnavailableError = (err) => {
+  if (err.statusCode === 404) {
+    // Older Management API version where the endpoint is not available.
+    return true;
+  }
+  if (err.statusCode === 403
+    && err.originalError
+    && err.originalError.response
+    && err.originalError.response.body
+    && err.originalError.response.body.errorCode === 'hooks_not_allowed') {
```

Comment:
Being defensive here, because I suspect this endpoint is enabled in all environments, but this condition will make it 100% safe (ex. even in older PSaaS environments that have not been patched in a long time).